docs/pipelines/release/artifacts.md

```diff
@@ -340,6 +340,12 @@ The following features are available when using Azure Artifacts sources:
 
 <p />
 
+*Note:* when obtaining Maven artifacts and the artifact is a snapshot build, mulitple versions of that snapshot may be downloaded at once (e.g. `myApplication-2.1.0.BUILD-20190920.220048-3.jar`, `myApplication-2.1.0.BUILD-20190820.221046-2.jar`, `myApplication-2.1.0.BUILD-20190820.220331-1.jar`). You will likely need to add additional automation to keep only the latest artifact prior to subsequent deployment steps. This can be accomplished with the following PowerShell snippet:
```

Comment:
This looks great and thank you for the input! The only thing I would say is note to use the Note, since you have a PowerShell snippet to copy I would let it be a section on it's own. 
However, if you feel that a 'Note' here is better, use the format:
> [!NOTE]
> <Your note string here.>

Reply:
Replaced with a subheading.